templates/2.0/mapping.json

```diff
@@ -25,7 +25,7 @@
             "ip": {
                 "section": "cloud_agent",
                 "option": "cloudagent_ip",
-                "default": "127.0.0.1"
+                "default": "0.0.0.0"
```

Comment:
IMHO that is not correct. The default, 127.0.0.1, is more secure as there is control of the interface that will be attached.  Using 0.0.0.0 by default will attach the service into all interfaces by default, a much less secure one.

As a disclosure, I am using 0.0.0.0 in the RPM package so the user can star the service soon, but it is expected that the user will change it to a more secure option.

Reply:
I second @aplanas evaluation here. Thanks to both our `keylime_upgrade_config` tool as well as the ability to override values on configuration files from environment variables, I simply cannot see the need for making the **less** secure option (i.e., listen on all interfaces) the default on the template.